daffodil-core/src/main/scala/org/apache/daffodil/core/dsom/DFDLSchemaFile.scala

```diff
@@ -45,16 +46,38 @@ class DFDLSchemaFileLoadErrorHandler(schemaFileLocation: SchemaFileLocation)
   private def loaderErrors = loaderErrors_
   private def loaderWarnings = loaderWarnings_
 
-  private def loaderSDEs: Seq[Diagnostic] = loaderErrors.map {
-    new SchemaDefinitionError(schemaFileLocation, "Error loading schema due to %s", _)
+  private def loaderSDEs: Seq[Diagnostic] = loaderErrors.map { err =>
+    val errMessage = err.getMessage
+    // we create a new SchemaFileLocation because the Xerces error has line and column info that
+    // the original schemaFileLocation that's passed in doesn't contain, so we create this more
+    // complete SchemaFileLocation
+    val xsfl = new XercesSchemaFileLocatable(err, "")
```

Comment:
That was intentional. It represents the diagnostic debug name, which we changed from "Schema File" to an empty string

Reply:
I see, makes sense. So does it always want to be empty string then? Maybe instead of it being a parameter XercesSchemaFileLocatable can do something like  `override diagnosticDebugName =  ""` with a comment explaining why that's preferred over "Schema File"?